microsoft/testsuites/network/networksettings.py

```diff
@@ -549,8 +551,10 @@ def validate_device_msg_level_change(self, node: Node, log: Logger) -> None:
             Steps:
             1. Get all the device's statistics.
             2. Validate device statistics lists per queue statistics as well.
-            3. Run traffic using iperf3 and validate traffic is evenly distributed
-                among different VMBUS channel queues.
+            3. Run traffic using iperf3 and check stats for each device.
+            4. if the same queue (say queue #0) is always inactive among 5 test runs,
+                and the number of connections is >= 4 (total #queues), test should fail
```

Comment:
1. What's "the number of connections"? I cannot find it in the code logic. If there are concept used, the term should be consistent.
2. I feel there is no difference between run 1 or 5 times. If it's not enough to run iperf3 with 10 seconds, let's just extend it to 20 seconds.
3. Because I don't understand the connection or queue here, so I cannot understand the reason it's considered failed.

Reply:
1. The number of connections here means the number of VMBUS channels i.e. the logic holds good if there are at least 4 tx/rx queue counters. For eg - 
                            tx_queue_0_packets/rx_queue_0_packets
                            tx_queue_1_packets/rx_queue_1_packets
                            tx_queue_2_packets/rx_queue_2_packets
                            tx_queue_3_packets/rx_queue_3_packets

2. Run iperf 5 times so we can see if for those 5 runs, one of the queues is completely starved off of packets.

Reply:
1. I can change the wording. if count of channels is lower than 4 then as per Haiyang (will fwd you email), even if one queue is tmtimg/rcvng 0 packets, that might still be ok since there are less number of queues/channels.

2. The difference is that - if more number of packets are going through then there is higher chance for each queue to tx/tx packets as compared to if there are smaller number of packets. Yeah we can increase time but I think what we would want to do is check the stats at some interval and make sure that each queue is tx'ing/rxlng some packets.